orange/src/main/java/com/hartwig/hmftools/orange/OrangeConfig.java

```diff
@@ -345,30 +360,49 @@ static OrangeConfig createConfig(final ConfigBuilder configBuilder) {
         String lilacCoverage = Config.fileIfExists(LilacAllele.generateFilename(lilacDir, tumorSampleId));
         String lilacQc = Config.fileIfExists(LilacQcData.generateFilename(lilacDir, tumorSampleId));
 
-        String virusDir = getToolDirectory(configBuilder, pipelineSampleRootDir, sampleDataDir, VIRUS_DIR_CFG, VIRUS_BREAKEND_DIR);
-        String virusAnnotations =
-                virusDir != null ? Config.fileIfExists(AnnotatedVirusFile.generateFileName(virusDir, tumorSampleId)) : null;
+        String tumorMetricsFile = getWgsMetricsFile(configBuilder, TUMOR_SAMPLE_WGS_METRICS_FILE, tumorSampleId, pipelineSampleRootDir);
+        String tumorFlagstatFile = getFlagstatFile(configBuilder, TUMOR_SAMPLE_FLAGSTAT_FILE, tumorSampleId, pipelineSampleRootDir);
 
-        String chordDir = getToolDirectory(configBuilder, pipelineSampleRootDir, sampleDataDir, CHORD_DIR_CFG, CHORD_DIR);
-        String chordPredictions = chordDir != null ? Config.fileIfExists(ChordDataFile.generateFilename(chordDir, tumorSampleId)) : null;
+        String virusAnnotations = null;
+        String chordPredictions = null;
+        String cuppaDataFile = null;
+        String cuppaSummaryPlot = null;
+        String cuppaFeaturesPlot = null;
+        String cuppaChartPlot = null;
+        String sigAllocations = null;
+        String peachGenotype = null;
+        String refMetricsFile = null;
+        String refFlagstatFile = null;
 
-        String cuppaDir = getToolDirectory(configBuilder, pipelineSampleRootDir, sampleDataDir, CUPPA_DIR_CFG, CUPPA_DIR);
-        String cuppaDataFile = cuppaDir != null ? Config.fileIfExists(CuppaDataFile.generateFilename(cuppaDir, tumorSampleId)) : null;
-        String cuppaSummaryPlot =
-                cuppaDir != null ? Config.fileIfExists(CuppaDataFile.generateReportSummaryPlotFilename(cuppaDir, tumorSampleId)) : null;
-        String cuppaFeaturesPlot = cuppaDir != null
-                ? Config.optionalFileIfExists(CuppaDataFile.generateReportFeaturesPlotFilename(cuppaDir, tumorSampleId))
-                : null;
-        String cuppaChartPlot =
-                cuppaDir != null ? Config.fileIfExists(CuppaDataFile.generateChartPlotFilename(cuppaDir, tumorSampleId)) : null;
+        if(refSampleId != null)
```

Comment:
I understand in practice this will be fine assuming we never do WGS without reference, but still think we should consider modeling the 2 properties separately:
 - tumorOnlyMode yes/no
 - experimentType TARGETED or WHOLE_GENOME

In OrangeAlgo we pull the experiment type from purple but imagine that's ugly inside the config

```
ExperimentType experimentType = purpleData.purityContext().targeted() ? ExperimentType.TARGETED : ExperimentType.WHOLE_GENOME;
```

Reply:
No sure if this was a mistyping - we do run WGS tumor-only mode ie not for panels, so then the ref ID will be null.

Over email I suggested that for now we don't make a distinction between tumor-only and panel modes, since the behaviour for Orange will be the same. I think that is still the case, but do you know of any differences? If so, I should be handling them in this branch/task.

Reply:
I believe in the pipeline we do run virus breakend and virus interpreter for tumor-only WGS. In your current solution this data won't be loaded up in ORANGE, this seems inconsistent (I am not 100% sure I am correct, didn't check pipeline code).